WebLib/Models/Author.cs

```diff
@@ -0,0 +1,116 @@
+﻿// <copyright file="Author.cs" company="My company">
+//     Copyright (c) My company". All rights reserved.
+// </copyright>
+
+namespace WebApi.Models
+{
+    using System;
+    using System.ComponentModel.DataAnnotations;
+
+    /// <summary>
+    /// Author model class
+    /// </summary>
+    public class Author
+    {
+        /// <summary>
+        /// Authors static counter for make each author unique
+        /// </summary>
+        private static int authorIDCount;
+
+        /// <summary>
+        /// Initializes static members of the <see cref="Author"/> class
+        /// </summary>
+        static Author()
+        {
+            authorIDCount = 1;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Author"/> class
+        /// </summary>
+        /// <param name="surname">Surname string</param>
+        /// <param name="name">Name string</param>
+        /// <param name="patronymic">Patronymic string</param>
+        public Author(string surname, string name, string patronymic)
+        {
+            this.Surname = surname;
+            this.Name = name;
+            this.Patronymic = patronymic;
+            this.AuthorID = authorIDCount++;
+        }
+
+        /// <summary>
+        /// Gets author id
+        /// </summary>
+        public int AuthorID
+        {
+            get;
+            private set;
+        }
+
+        /// <summary>
+        /// Gets surname of author
+        /// </summary>
+        [Required(ErrorMessage = "Every author has surname name!")]
```

Comment:
Do you ever use this message? I suppose not. Check in google how to return in response data annotation error messages.